src/components/accordion/accordionReducer.js

```diff
@@ -0,0 +1,33 @@
+// @flow strict
+
+export type ActionType = {
+  type: 'accordion/SET_OPENED',
+  payload: {id: string, value: boolean},
+};
+
+export type StateType = $ReadOnly<{
+  opened: {
+    [key: string]: boolean,
+  },
+  noGapBetweenElements: boolean,
+  allowMultiple: boolean,
```

Comment:
let's not sync our props with state. Moving reducer to the component will help resolving the issue.